cnv/cnv_users_guide/cnv-installing-virtio-drivers-on-new-windows-vm.adoc

```diff
@@ -0,0 +1,21 @@
+[id="cnv-installing-virtio-drivers-on-new-windows-vm"]
+= Installing VirtIO driver on a new Windows virtual machine
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-installing-virtio-drivers-on-new-windows-vm"
+toc::[]
+
+.Prerequisites
+
+* Windows installation media accessible by the virtual machine.
```

Comment:
It can be done by adding the windows image as a CDROM. Usually as a SATA bus.
```yaml
apiVersion: kubevirt.io/v1alpha3
kind: VirtualMachineInstance
metadata:
  labels:
    special: vmi-windows
  name: vmi-windows
spec:
  domain:
    devices:
      disks:
      - disk:
          bus: sata
        name: pvcdisk
      - cdrom:
          bus: sata
        name: wininstall
  volumes:
  - name: pvcdisk
    persistentVolumeClaim:
      claimName: disk-windows
  - name: wininstall
    persistentVolumeClaim:
      claimName: disk-windows-install
```

Do you have a guide somewhere you can reference? In my opinion it would be better than just adding it here.

Reply:
@petrkotas To get the Windows installation media in the pvc, do we use the same concept as documented here?
https://docs.openshift.com/container-platform/3.11/cnv_users_guide/cnv_users_guide.html#importvm-datavolumes
If so, I can xref to that once it has been migrated (in progress)

Reply:
@aburdenthehand that is a good find. Yes you can do it that way. You can also populate the persistent volume manually, but in my opinion we should not point that in the guide.

Reply:
@petrkotas - I've linked that prerequisite to the `importvm-datavolumes` content linked in the earlier comment. That content is still being updated+migrated but I've included the anchor of what it will be called.

@kalexand-rh I believe this is the only outstanding issue with this PR. If this looks okay then it's ready to merge.